unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -580,14 +581,22 @@ protected void CopyFrom(AvatarRenderer original)
             this.eyesController = original.eyesController;
         }
 
-        public void SetVisibility(bool newVisibility)
+        public void SetGOVisibility(bool newVisibility)
         {
             //NOTE(Brian): Avatar being loaded needs the renderer.enabled as false until the loading finishes.
             //             So we can' manipulate the values because it'd show an incomplete avatar. Its easier to just deactivate the gameObject.
             if (gameObject.activeSelf != newVisibility)
                 gameObject.SetActive(newVisibility);
         }
 
+        public void SetCombinedMeshVisibility(bool newVisibility)
```

Comment:
As stated before, I would change this to `SetRendererEnabled` and use a property for when the GPU Skinning is there.

Reply:
renamed!